genai-perf/tests/test_image_retrieval_profile_data_parser.py

```diff
@@ -43,6 +43,7 @@ def check_image_retrieval_metrics(
     assert m1.request_throughputs == pytest.approx(m2.request_throughputs)
     assert m1.image_latencies == m2.image_latencies
     assert m1.image_throughputs == pytest.approx(m2.image_throughputs)
+    assert m1.request_goodputs is None or m1.request_goodputs == m2.request_goodputs
```

Comment:
Why is there an or here? Shouldn't only one assertion be true?

Reply:
This is for existing tests with no goodput computation. I will try to find a way to get rid of this